subsys/net/ip/net_shell.c

```diff
@@ -4084,7 +4088,7 @@ static int ping_ipv4(const struct shell *shell,
 #define remove_ipv4_ping_handler()
 #endif /* CONFIG_NET_IPV4 */
 
-static int parse_arg(size_t *i, size_t argc, char *argv[])
+static int parse_arg(size_t *i, size_t argc, char *argv[], bool allow_hex)
```

Comment:
Do we really need this flag? We can allow HEX setting for options, no harm right?

Reply:
Yeah, on second thought I think you're right.